README.md

```diff
@@ -1,9 +1,102 @@
 # Pendulum Chain
 
-Pendulum chain by SatoshiPay.
+Pendulum chain by SatoshiPay. More information about Pendulum can be found [here](https://pendulum.gitbook.io/pendulum-docs/).
 
-Node runtime that is parachain ready (configured with cumulus).
+### How to Run Tests
+To run the unit tests, execute
+```
+cargo test
+```
 
-🚧 This is a work in progress. 🚧
+### How to Build
+To build the project, execute
+```
+cargo b --release
+```
+A successful build will create a **target** folder with a sub folder **release**.
+Check for `parachain-collator` in the **release** folder.
 
-Based on Substrate. Repository based on [Substrate parachain template](https://github.com/substrate-developer-hub/substrate-parachain-template).
+### How to Generate Chain Spec
+There are 2 different [runtime](runtime)s currently in the repo; **amplitude** for the Amplitude parachain and **development** for the Pendulum parachain. Any of these runtimes are used depending on the config. The config is created by generating the chain spec:
+```
+./target/release/parachain-collator build-spec --disable-default-bootnode > local-parachain-plain.json
+```
+To create the amplitude spec, the `--chain` has to be specified:
+```
+./target/release/parachain-collator build-spec --chain amplitude --disable-default-bootnode > local-parachain-plain.json
+```
+
+For the raw chain spec, just add the option `--raw` and the `--chain` should be the one you just generated:
+```
+./target/release/parachain-collator build-spec --chain local-parachain-plain.json --raw --disable-default-bootnode > local-parachain-raw.json
+```
+
+### How to Generate Wasm:
+```
+/target/release/parachain-collator export-genesis-wasm --chain local-parachain-raw.json > para-2000-wasm
+```
+### How to Generate Genesis State:
+```
+./target/release/parachain-collator export-genesis-state --chain rococo-local-parachain-raw.json > para-2000-genesis
+```
+
+Note: The amplitude chain specs, the wasm and the genesis state are already available in the [res](res) folder.
+
+### How to Run:
+To run the collator, execute:
+```
+./target/release/parachain-collator
+--collator \
+--allow-private-ipv4 \
+--unsafe-ws-external \
+--rpc-cors all \
+--rpc-external \
+--rpc-methods Unsafe \
+--name <INSERT_NAME> \
+--ws-port <WS_PORT> --port <PARA_PORT> --rpc-port <RPC_PORT> \
```

Comment:
I think is good to have this knd of variables like WS_PORT. But we also would need to explain what they mean.
Also, idk what `PARA_PORT` means, but `--port` is used as the _peering port_ ,  to communicate with other nodes.

Reply:
I added a table to define them